### PR TITLE
(SIMP-3797) Changelog Update

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Updated the Changelog with the user-relevant changes. Items that would
not affect the user from an operational point of view were not
mentioned.

SIMP-3797 #close

### CODE REVIEW

File: Changelog.rst [hunk 1]
```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
...
-* Added as a supported simp extra
+* Added as a SIMP extra
 
-pupmod-puppet-gitlab
-^^^^^^^^^^^^^^^^^^^^^^^
-* Added as a simp extra
+pupmod-simp-selinux
+^^^^^^^^^^^^^^^^^^^
+* Added a reboot notification on appropriate SELinux state changes
+* Ensure that a ``/.autorelabel`` file is created on appropriate SELinux state
```
Comment (jeefberkey) [inline Changelog.rst]: This was set to be off by default because jankiness

File: Changelog.rst [hunk 2]
```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
...
+  new ``auth.conf``, via the ``puppet_authorization::rule`` defined type
 * The stock rules are managed in ``pupmod::master::simp_auth``
 
-RPM Installation
-^^^^^^^^^^^^^^^^
+No Longer Delivering ClamAV DAT Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Given the wide spacing of SIMP releases, the team determined that it was
+effective for us to maintain the ``simp-rsync-clamav`` RPM with upstream ClamAV
```
Comment (lnemsick-simp) [inline Changelog.rst]: effective -> ineffective?

File: Changelog.rst [hunk 3]
```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
...
-Puppet Enterprise SIMP
-""""""""""""""""""""""
+We have re-added SNMP support after a thorough re-assessment and update from
+our legacy ``snmp`` module. We now build upon a community module and wrap the
+SIMP-specific components on top of it.
+
+Preparing for Puppet 5
+^^^^^^^^^^^^^^^^^^^^^^
+
+We have been updating all of our modules to include tests for Puppet 5 and, so
```
Comment (lnemsick-simp) [inline Changelog.rst]: I do not think this statement is appropriate:
* Not all modules are testing with puppet 5.X in TravisCI
* Most modules have the puppet version < 5.0.0 in their metadata.json files.  So, these modules cannot be plunked down on a system running puppet 5.

File: Changelog.rst [hunk 4]
```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
...
+far, things have gone quite well. Many module version bumps were simply due to
+the fact that Puppet 5 tests were added and no functionality was changed.
```
Comment (lnemsick-simp) [inline Changelog.rst]: Not really true.  Some module bumps were for non-breaking dependency version bumps, like concat 3.0.0

File: Changelog.rst [hunk 5]
```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
...
 +---------------------+-------------+-------------+
 
+
 Removed Modules
 ---------------
 
+pupmod-herculesteam-augeasproviders_base
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Has internal bugs and was not in use by any SIMP components
+
```
Comment (lnemsick-simp) [inline Changelog.rst]: Also removed augeasproviders meta-module

File: Changelog.rst (hunk 3 above)
Comment (trevor-vaughan) [reply Changelog.rst]: @lnemsick-simp I'll change it to 'in progress'

File: Changelog.rst [hunk 6]
```diff
@@ -44,69 +45,241 @@ Puppetserver auth.conf
...
+At this time, we expect that the update to Puppet 5 will be seamless for
+existing SIMP 6 installations.
 
-FOSS SIMP
-"""""""""
+Non-Breaking Version Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``sudo yum -y install simp-adapter simp``
+Many modules had dependencies that were updated in a manner that was breaking
```
Comment (lnemsick-simp) [inline Changelog.rst]: This is a mouthful.  Is there a simpler way to state this?

Comment (trevor-vaughan) [reply Changelog.rst]: @lnemsick-simp Probably...patches very welcome on wording.

**Comment by trevor-vaughan** [comment]
Rebased

**Pull request merged**